internal/v1/api.yaml

```diff
@@ -1077,6 +1077,11 @@ components:
           description: |
             List of custom repositories including all the repository details needed in order
             to recreate the repositories.
+        snapshot_date:
+          type: string
+          description: |
+            Importing the snapshot date will not yet be supported. It is exported for informative reasons.
+            The format is YYYY-MM-DD.
```

Comment:
Isntt there a date type in OpenAPI JSON Schema?

Reply:
Might be, but I would probably leave this in the same format that we have in the image requests `snapshot_date` field for consistency.